itests/utilities/transfer_utilities/transfer_tx_utility.go

```diff
@@ -92,3 +140,20 @@ func GetNewAccountWithFunds(suite *f.BaseSuite, version byte, scheme proto.Schem
 	require.NoError(suite.T(), err)
 	return accNumber
 }
+
+// amount of Asset that transfered from one account to another, by default it will be all amount of Asset
+func TransferAssetAmount(suite *f.BaseSuite, version byte, scheme proto.Scheme, assetId crypto.Digest,
+	from, to int, assetAmount ...uint64) {
```

Comment:
Are you sure that `assetAmount` is necessary here? I don't see any usages of `TransferAssetAmount` where `assetAmount` argument is not empty.

Reply:
It will be used a little bit later in more complicated cases